shared/constants/network.ts

```diff
@@ -177,6 +179,7 @@ export const HARMONY_DISPLAY_NAME = 'Harmony Mainnet Shard 0';
 export const PALM_DISPLAY_NAME = 'Palm';
 export const AURORA_DISPLAY_NAME = 'Aurora Mainnet';
 export const CELO_DISPLAY_NAME = 'Celo Mainnet';
+export const EOS_DISPLAY_NAME = 'EOS EVM Network';
 
```

Comment:
**Optional**  
I would drop "Network" here. Our branding just calls it EOS EVM, no "network" suffix.
https://eosnetwork.com/eos-evm

Reply:
Updated ✅  to "EOS EVM"

Reply:
Thank you! I don't have permissions to mark this thread "resolved" I guess, but...resolved!